### ISSUE TITLE
Default key assignment

### ISSUE THREAD

**Issue by qwertychouskie** [supertuxkart/stk-code#3925]
As is already known, often time keyboards have trouble with multiple keys.  When I run into this issue, something that always seems to work for me is to shift the key assignments two keys to the left, so V is Nitro, C is look back, and X is skid.  I don't think I've run into a keyboard where this layout didn't work.  Maybe we should look into changing the default assignment.

**Comment by Alayan-stk-2** [comment]
My keyboard configurations are different from the default one, and more comfortable in my opinion, but it's hard to objectively assess what is the best default... and the default config should already not run in "keyboards are evil" issues ?

**Comment by qwertychouskie** [comment]
> and the default config should already not run in "keyboards are evil" issues ?

Agreed it shouldn't, but unfortunately it does.  I can think of any systems where my config failed, however, and it uses the same finger layout (which is why I picked it in the first place), so it should be a relatively safe change.

**Comment by deveee** [comment]
I had issues with left arrow on every laptop and every keyboard on which I was playing STK.

And personally I use:
- left control - fire
- z - nitro
- x - skidding
- a - look back
- s - rescue

and I never had any problems with this configuration.
